src/AndcultureCode.CSharp.Sitefinity.Core/App.config

```diff
@@ -0,0 +1,48 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+	<appSettings>
```

Comment:
Nitpick: It looks like there are some indentation inconsistencies in this config file.